client/shuffler/src/lib.rs

```diff
@@ -1,17 +1,29 @@
 #![cfg_attr(not(feature = "std"), no_std)]
+use sp_api::Encode;
+use sp_api::HashT;
+
+use sp_runtime::traits::{BlakeTwo256, Block as BlockT};
+use sp_runtime::AccountId32;
+use sp_std::collections::btree_map::BTreeMap;
+use sp_std::convert::TryInto;
+
+use sp_std::collections::vec_deque::VecDeque;
+use sp_std::vec::Vec;
+
+#[cfg(feature = "std")]
```

Comment:
Why do we need std here and later and will it work in WASM?

Reply:
this pallet needs to be compatible with/withour wasm because we are using it from the native part (block_budiled) and from inside the runtime(Executive pallet). So above is used to make it compatible with both. Its common approach from what i saw in other crates